README.md

```diff
@@ -14,9 +14,7 @@ This repository contains idiomatic C++ client libraries for the following
 [Google Cloud Platform](https://cloud.google.com/) services.
 
 > Please check the [CHANGELOG] for important announcements and upcoming
-> changes.<br>
-> In particular, note that on (or about) **2022-06-01** this library will
-> require C++ >= 14.
+> changes.
```

Comment:
uber nit: I think this will fit on one line now.

Reply:
I believe we concluded during the `mdformat` work that we weren't going to worry about filling markdown lines.